dashboard/src/containers/RoutesContainer/Routes.test.tsx

```diff
@@ -58,9 +59,9 @@ it("should render a redirect to the login page", () => {
     <StaticRouter location="/" context={{}}>
       <Routes
         {...emptyRouteComponentProps}
-        cluster={"default"}
-        namespace={""}
-        authenticated={true}
+        cluster={""}
+        currentNamespace={""}
+        authenticated={false}
```

Comment:
I'm confused what this test is now testing compared to the following? It looks like they are both testing that the redirect is rendered when unauthed (and that the value of the state should be irrelevant - so only one test required?)

Reply:
yes, the only difference is that the `cluster` and `currentNamespace` info is populated, which should be ignored. Updating the test description for being more clear.